recipes/dcmstack/meta.yaml

```diff
@@ -0,0 +1,34 @@
+{% set version = "v0.6.2" %}
+
+package:
+  name: dcmstack
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/moloney/dcmstack.git
+  git_tag: {{ version }}
```

Comment:
Again, this is based on the example provided. I assumed that should be the default. I am happy to switch to the github release tarball, if that's preferred (I'd also recommend changing the example accordingly as well, if that's the preferred format).